client/src/common/constants/environment.ts

```diff
@@ -0,0 +1,5 @@
+const ENV = {
+  backendUrl: import.meta.env.VITE_BACKEND_URL ?? ''
+};
+
+export default ENV;
```

Comment:
There is a better way of loading environments than using a constants file, I have done it in my staff front-end. I will PR it today so we can get those extra goodies I built into the app.

I noticed that this is unused, so I recommend that you just delete it.

Reply:
I see how youre doing it in `main` I do like the setting up the axios instance like that, but the env is the same